ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/core/TransactionReceipt.java

```diff
@@ -42,25 +46,37 @@
   private final LogsBloomFilter bloomFilter;
   private final int status;
   private final TransactionReceiptType transactionReceiptType;
+  private final Optional<String> revertReason;
 
   /**
    * Creates an instance of a state root-encoded transaction receipt.
    *
    * @param stateRoot the state root for the world state after the transaction has been processed
    * @param cumulativeGasUsed the total amount of gas consumed in the block after this transaction
    * @param logs the logs generated within the transaction
+   * @param revertReason whether storing the revert reason is for failed transactions
```

Comment:
Grammer.

`@param revertReason whether to store the revert reason for failed transactions`

Reply:
This is actually the revert reason not a flag whether or not to store the revert reason. So have updated this and the below constructor to reflect this.